src/analysis/normed_space/basic.lean

```diff
@@ -288,20 +285,18 @@ nnreal.eq $ norm_mul a b
 /-- `nnnorm` as a `monoid_with_zero_hom`. -/
 @[simps] def nnnorm_hom : α →*₀ ℝ≥0 := ⟨nnnorm, nnnorm_zero, nnnorm_one, nnnorm_mul⟩
 
+lemma list.norm_prod (l : list α) : ∥l.prod∥ = (l.map norm).prod :=
+(norm_hom.to_monoid_hom : α →* ℝ).map_list_prod _
+
+lemma list.nnnorm_prod (l : list α) : ∥l.prod∥₊ = (l.map nnnorm).prod :=
```

Comment:
For the sake of minimizing the diff, can you leave these lemmas in the same place as the old prod lemmas?